api/test/geo-features.e2e-spec.ts

```diff
@@ -49,6 +50,10 @@ describe('GeoFeaturesModule (e2e)', () => {
   const country = 'NAM';
   const l1AdminArea = 'NAM.13_1';
   const l2AdminArea = 'NAM.13.5_1';
+  const geoFeaturesFilters = {
+    cheeta: { featureClassName: 'iucn_acinonyxjubatus', alias: 'cheetah' },
```

Comment:
is this mapping specific / well known? Where `featureClassName` comes from?

Reply:
this may come from source data from the ETL pipelines; its format and content may depend on the data source